content/docs/setup/kubernetes/multicluster-install/index.md

```diff
@@ -267,3 +277,57 @@ $ kubectl delete -f $HOME/istio-remote.yaml
 {{< text bash >}}
 $ helm delete --purge istio-remote
 {{< /text >}}
+
+## Deployment considerations
+The above procedure provides a simple and step by step guide to deploy a multicluster
+environment.  A production environment might require additional steps or more complex
+deployment options.  The procedure gathers the endpoint IPs of Istio services and uses
+them to invoke Helm. This create Istio services on the remote clusters. As part of  
+creating those services and endpoints in the remote cluster Kubernetes will 
+add DNS entries into kube-dns.  This allows kube-dns in the remote clusters to  
+resolve the Istio service names for all envoy sidecars in those remote clusters.
+Since Kubernetes pods don't have stable IPs, restart of any Istio service pod in
+the control plane cluster will cause its endpoint to be changed. Therefore, any
+connection made from remote clusters to that endpoint will be broken. This is 
+documented in [istio/istio#4822](https://github.com/istio/istio/issues/4822)
+
+There are a number of ways to either avoid or resolve this scenario. This section 
+provides a high level overview of these options.   
+
+* Update the DNS entries 
+* Use a load balancer service type 
+* Expose the Istio services via a gateway
+
+### Update the DNS entries
+
+Upon any failure or pod restart kube-dns on the remote clusters can be
+updated with the correct endpoint mappings for the Istio services.  There 
+are a number of ways this can be done. The most obvious is to rerun the Helm 
+install in the remote cluster after the Istio services on the control plane 
+cluster have restarted. 
+
+### Use 'LoadBalance' service type
```

Comment:
LoadBalance/LoadBalancer - can remove `` - should be in .spelling file

Reply:
Tim had asked for the back ticks so it was more clearly a kubernetes resource